deploy_notes_initial.md

```diff
@@ -16,13 +16,18 @@ These steps are intended to be run the first time a development or production co
       1. Using the SSH protocol: `git clone git@github.com:codeforgreenville/upstate_tech_cal_service.git`
       1. Or, using the HTTPS protcol: `git clone https://github.com/codeforgreenville/upstate_tech_cal_service.git`
    6. `cd upstate_tech_cal_service`
-   7. [Install Miniconda](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links) (Linux example) `curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh`
-   8. `sh miniconda.sh -b -p $HOME/miniconda`
-   9. `export PATH="$HOME/miniconda/bin:$PATH"`
-   10. `conda env create -f environment.yml`
-   1. Activate the application environment
-      1. `conda activate cal_service`
-      1. Or, the older way will work too: `source activate cal_service`
+   7. [Install Pipenv](https://pipenv.pypa.io/en/latest/#install-pipenv-today)  
+   For Linux: `curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python`  
+      1. Verify the installation with `pipenv --version`, the output should look something like:  
```

Comment:
I always avoid using a single bullet point, which is a technical writing best practice.

If I don't have enough content for a bullet point I'll usually combine the top content with what would be a single bullet by separating them with a dash, like 

[Install Pipenv](https://pipenv.pypa.io/en/latest/#install-pipenv-today)  - Verify the installation with `pipenv --version`, the output should look something like:

Reply:
Mean to say 

> If I don't have enough content for two or more bullet points

Reply:
Do you have a reference for technical writing? It'd be really useful for me to bookmark and have on-hand.

Reply:
Thanks. The bullet point thing is probably the only thing I remember from years ago. No resources in mind, but I'm sure there are some best practices lists out there.